automation/vars/default.yaml

```diff
@@ -137,15 +149,27 @@ vas:
             src_file: service-values.yaml
         build_output: control-plane.yaml
 
-      - path: examples/va/nfv/ovs-dpdk/edpm
+      - path: examples/va/nfv/ovs-dpdk/edpm/nodeset
         wait_conditions:
           - >-
             oc -n openstack wait
-            osdpd edpm-deployment --for condition=Ready --timeout=1200s
+            osdpns openstack-edpm --for condition=SetupReady
+            --timeout=30m
```

Comment:
```suggestion
            --timeout=10m
```

Reply:
@fao89 IIUC this waits for the NodeSet to be ready, that's having the target nodes of the deployment UP and reachable. In fully virtualized environment 10 mins are more than enough, but based on my testing in real HW, the needed time is near 25 mins, 30 mins looks more than fine for a real Metal3 provisioning scenario.

Reply:
actually, `SetupReady` is faster than `Ready`, because it does not depend on the deployment.
- `Ready`: when the deployment finishes - https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/controllers/openstackdataplanenodeset_controller.go#L425-L428
- `SetupReady`: when the NodeSet is parsed and the inventory secret is generated - https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/controllers/openstackdataplanenodeset_controller.go#L387-L403

Reply:
sorry, my install_yamls bias got the provisioning for granted 😅

Reply:
we need to fix HCI then:
https://github.com/openstack-k8s-operators/architecture/blob/main/automation/vars/default.yaml#L27-L32
https://github.com/openstack-k8s-operators/architecture/blob/main/automation/vars/default.yaml#L54-L59

Reply:
actually not, HCI is pre-provisioned